src/main/scala/org/graphframes/lib/ShortestPaths.scala

```diff
@@ -28,16 +28,17 @@ import org.graphframes.GraphFrame
 /**
  * Computes shortest paths to the given set of landmark vertices.
  *
- * @param graph the graph for which to compute the shortest paths
- * @return a graph where each vertex attribute is a map containing the shortest-path distance to
- * each reachable landmark vertex.
+ * The returned vertices DataFrame contains one additional column:
+ *  - distances (MapType[vertex ID type, IntegerType]): For each vertex v, a map containing
```

Comment:
nit: some comments use sql-ish syntax for the type (like above), while some others use scala-ish syntax (`Array[Double]`). Any preference?